fetch.bs

```diff
@@ -197,6 +197,11 @@ of:
 <a>queue a fetch task</a> on <var>request</var> to <a>process request end-of-body</a>
 for <var>request</var>.
 
+<p>To <dfn>serialize an integer</dfn>, represent it as a DOMString of the shortest possible decimal
```

Comment:
s/DOMString/string/ please. I don't like using IDL types (and there's an issue against IDL about mapping DOMString to JavaScript string once it enters the specification side).